__tests__/App.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import App from '../client/src/components/App.jsx';
+
+import { shallow, mount, render } from 'enzyme';
+
+
+describe('App component', () => {
+
+    test('should render the <App /> component', () => {
+        const wrapper = shallow( <App /> );
+    })
```

Comment:
Looks like this test was missing an expect statement

```suggestion
        const wrapper = shallow( <App /> );
        expect(wrapper.exists());
    })
```